src/main/java/com/notably/logic/parser/OpenCommandParser.java

```diff
@@ -34,12 +34,14 @@ public OpenCommandParser(Model notablyModel) {
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_TITLE);
 
+        String titles;
         if (!ParserUtil.arePrefixesPresent(argMultimap, PREFIX_TITLE)
                 || !argMultimap.getPreamble().isEmpty()) {
-            throw new ParseException(String.format("Invalid input"));
+            titles = args.trim();
```

Comment:
Let's check for an empty title. That is, throw a `ParseException` if the user types `open` only, without any arguments!

Reply:
done!